.gitignore

```diff
@@ -1 +1,7 @@
-node_modules/
+# Lot of node developers have `node_modules` in `~/.gitignore`
+# so we use this hack to reinable it than disable everything
```

Comment:
Nit: it's not really a hack, but part of the way .gitignore is supposed to work. NBD, though. Merge it!